src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        startDuke();
+
+        Scanner userInput = new Scanner(System.in);
+        String line;
+        ArrayList<Task> tasks = new ArrayList<>();
+        Storage.storageSetup(tasks);
+        line = userInput.nextLine();
+        while (!"bye".equals(line)) {
```

Comment:
How about declaring a separate boolean name for this conditional? This will help [explain](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain) things better.